content/en/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -41,23 +43,30 @@ Administrators can specify a default `VolumeSnapshotClass` just for VolumeSnapsh
 that don't request any particular class to bind to.
 
 ```yaml
-apiVersion: snapshot.storage.k8s.io/v1alpha1
+apiVersion: snapshot.storage.k8s.io/v1beta1
 kind: VolumeSnapshotClass
 metadata:
   name: csi-hostpath-snapclass
-snapshotter: csi-hostpath
+driver: hostpath.csi.k8s.io 
+deletionPolicy: Delete
 parameters:
 ```
 
-### Snapshotter
+### Driver
 
-Volume snapshot classes have a snapshotter that determines what CSI volume plugin is
+Volume snapshot classes have a driver that determines what CSI volume plugin is
 used for provisioning VolumeSnapshots. This field must be specified.
 
+### DeletionPolicy
+
+Volume snapshot classes have a deletionPolicy. It enables an admin to configure what happens to a `VolumeSnapshotContent` when the `VolumeSnapshot` object it is bound to is to be deleted. The deletionPolicy of a volume snapshot can either be `Retain` or `Delete`. This field must be specified.
```

Comment:
Optional, nit level suggestion
```suggestion
Volume snapshot classes have a deletionPolicy. It enables you to configure what happens to a `VolumeSnapshotContent` when the `VolumeSnapshot` object it is bound to is to be deleted. The deletionPolicy of a volume snapshot can either be `Retain` or `Delete`. This field must be specified.
```

Really, objects like VolumeSnapshot should _not_ have backticks. That cleanup can wait until after the v1.17 release.

Reply:
Changed from "admin" to "you".